lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -67,4 +71,11 @@
       - "my_param <= 100"
       - "my_param >= 0"
     msg: "'my_param' must be between 0 and 100"
+
+- name: use quiet to avoid verbose output
+  assert:
+    that:
+      - "my_param <= 100"
+      - "my_param >= 0"
+    quiet: True
```

Comment:
```suggestion
    quiet: true
```
Ansible uses (or should use) only `true` and `yes` for boolean YAML values in the documentation.

Reply:
Done.